python/dgl/graphbolt/minibatch.py

```diff
@@ -500,7 +500,7 @@ def to_pyg_data(self):
             col_nodes = torch.cat(col_nodes)
             row_nodes = torch.cat(row_nodes)
             edge_index = torch.unique(
-                torch.stack((col_nodes, row_nodes)), dim=1
+                torch.stack((row_nodes, col_nodes)), dim=1
```

Comment:
why is this not captured by corresponding testcase?

Reply:
Because the test was also incorrect

Reply:
that makes sense...